docs/next.config.js

```diff
@@ -9,43 +10,14 @@ const { LANGUAGES, LANGUAGES_SSR } = require('./src/modules/constants');
 
 const workspaceRoot = path.join(__dirname, '../');
 
-/**
- * https://github.com/zeit/next.js/blob/287961ed9142a53f8e9a23bafb2f31257339ea98/packages/next/next-server/server/config.ts#L10
- * @typedef {'legacy' | 'blocking' | 'concurrent'} ReactRenderMode
- * legacy - ReactDOM.render(<App />)
- * legacy-strict - ReactDOM.render(<React.StrictMode><App /></React.StrictMode>, Element)
- * blocking - ReactDOM.createSyncRoot(Element).render(<App />)
- * concurrent - ReactDOM.createRoot(Element).render(<App />)
- * @type {ReactRenderMode | 'legacy-strict'}
- */
-const reactStrictMode = true;
-if (reactStrictMode) {
-  // eslint-disable-next-line no-console
-  console.log(`Using React.StrictMode.`);
-}
-
-const isDeployment = process.env.NETLIFY === 'true';
-
-module.exports = {
-  eslint: {
-    ignoreDuringBuilds: true,
-  },
+module.exports = withDocsInfra({
```

Comment:
Just to confirm I understand it correctly - was the issue here that without `withDocsInfra` the `DEPLOY_ENV` variable was missing and development `GOOGLE_ANALYTICS_ID` was used by default?

Reply:
@cherniavskii Yes, correct. I feel that at this point, it's not worth adding more checks as we don't market our docs as a standalone open source project and all the other docs are using `withDocsInfra` now.